docs/components/gantry/_index.md

```diff
@@ -34,338 +33,24 @@ Most robots with a gantry need at least the following hardware:
 
 ## Control your gantry with Viam's client SDK libraries
 
-Check out the [Client SDK Libraries Quick Start](/program/sdk-as-client/) documentation for an overview of how to get started connecting to your robot using these libraries.
-
-The following example assumes you have a gantry called "my_gantry" configured as a component of your robot.
-If your gantry has a different name, change the `name` in the example.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-```python
-from viam.components.gantry import Gantry
-
-async def main():
-    # Connect to your robot.
-    robot = await connect()
-
-    # Log an info message with the names of the different resources that are connected to your robot.
-    print('Resources:')
-    print(robot.resource_names)
-
-    # Connect to your gantry.
-    my_gantry = Gantry.from_robot(robot=robot, name='my_gantry')
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **code sample** tab, select your preferred programming language, and copy the sample code generated.
 
-    # Disconnect from your robot.
-    await robot.close()
+When executed, this sample code will create a connection to your robot as a client.
+Then control your robot programmatically by adding API method calls as shown in the following examples.
 
-if __name__ == '__main__':
-    asyncio.run(main())
-```
-
-{{% /tab %}}
-{{% tab name="Go" %}}
-
-```go
-import (
-  "context"
-
-  "github.com/edaniels/golog"
-
-  "go.viam.com/rdk/components/gantry"
-  "go.viam.com/rdk/referenceframe"
-)
-
-func main() {
-
-  // Create an instance of a logger.
-  logger := golog.NewDevelopmentLogger("client")
-
-  // Connect to your robot.
-  robot, err := client.New(
-      context.Background(),
-      "ADD YOUR ROBOT ADDRESS HERE. You can find this on the Code Sample tab of app.viam.com.",
-      logger,
-      client.WithDialOptions(rpc.WithCredentials(rpc.Credentials{
-          Type:    utils.CredentialsTypeRobotLocationSecret,
-          Payload: "ADD YOUR LOCATION SECRET HERE. You can find this on the Code Sample tab of app.viam.com",
-      })),
-  )
-
-  // Log any errors that occur.
-  if err != nil {
-      logger.Fatal(err)
-  }
-
-  // Delay closing your connection to your robot until main() exits.
-  defer robot.Close(context.Background())
-
-  // Log an info message with the names of the different resources that are connected to your robot.
-  logger.Info("Resources:")
-  logger.Info(robot.ResourceNames())
-
-  // Connect to your gantry.
-  myGantry, err := gantry.FromRobot(robot, "my_gantry")
-  if err != nil {
-    logger.Fatalf("cannot get gantry: %v", err)
-  }
-
-}
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+These examples assume you have a gantry called "my_gantry" configured as a component of your robot.
+If your gantry has a different name, change the `name` in the code.
 
 ## API
```

Comment:
I can't comment below out of diff but for DoCommand in the table (line 66) I noticed "Sends" should be "Send" for consistency.